### DISCUSSION TITLE
Duplicated entries should be disabled or marked as drafts by default

### DISCUSSION THREAD

**Discussion by MoritzLost** [craftcms/cms#10244] [Ideas] [answered]
When duplicating entries, the duplicated entry is _immediately activated and live_. From a user perspective, this is pretty much never what you want and expect. I can't imagine any scenario where it makes sense to have a live entry that is an exact duplicate of an existing entry with a `copy` suffix appended to the title. Instead, the most common use-case for duplicating an entry is to use it as a template for a new entry. In this case, you don't want the duplicated entry to be publicly visible until you've made some edits.

Of course you can disable the duplicated entry immediately so it's only public for a brief period of time. But if you're using hooks to do something with published entries, those hooks have already been triggered at that point. For example, we're using hooks to send push notifications and add new entries to an external search index. Both of those are triggered upon duplication of an entry.

I see two possible improvements for this:

- Make duplicated entries disabled by default.
- Make duplicated entries drafts instead, so they're in the same state you get when creating a new entry and saving it as a draft. This might be more in line with the regular editor experience for creating new entries.

**Comment by brandonkelly** [comment]
Already on the list for v4 😄  (Related to #9577)

**Update:** We weren’t able to get this into Craft 4 unfortunately, but it’s still planned.

**Reply by MoritzLost** [reply]
@brandonkelly Awesome, thanks!

**Comment by bencresty** [comment]
Completely agree with this suggestion. Had opened the same today (https://github.com/craftcms/cms/discussions/11986), but @brandonkelly reminded me you already had this one opened, so here I am :)

**Comment by brandonkelly** [comment, marked as answer]
We just got this in for Craft 5 (#12303). Going forward, entries created via “Save as a new entry” will start off in an unpublished draft state.

**Reply by brandonkelly** [reply]
Craft 5.0.0-alpha.5 is out with that change.

**Reply by cherrykoda** [reply]
Just bumped into this in a v4 site and glad to hear it! Good news for our client :)
